docs/application/web/api/6.5/device_api/mobile/tizen/metadata.html

```diff
@@ -80,9 +82,21 @@ <h2 id="method-summary">Summary of Interfaces and Methods</h2>
 <td><a href="#MetadataFileHandle">MetadataFileHandle</a></td>
 <td>
 <div>DOMString <a href="#MetadataFileHandle::get">get</a> (<a href="#MetadataType">MetadataType</a> type)</div>
+<div>
+<a href="filesystem.html#Blob">Blob</a> <a href="#MetadataFileHandle::getArtwork">getArtwork</a> ()</div>
+<div>
+<a href="filesystem.html#Blob">Blob</a> <a href="#MetadataFileHandle::getThumbnailFrame">getThumbnailFrame</a> ()</div>
```

Comment:
Just a question regarding https://code.sec.samsung.net/jira/browse/TWDAPI-271.

In the guide description, below are added
 - getArtwork()
 - getThumbnailFrame()
 - getFrameAtTime()
 - getSyncLyrics()

However from the list in the JIRA, I couldn't find getThumbnailFrame. Is there any reason for that?

Reply:
You are right, the name was changed during review, but I didn't update ACR description. I fixed description of https://code.sec.samsung.net/jira/browse/TWDAPI-271